content/v2.0/reference/cli/influx/secret/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: influx secret – Manage secrets
+description: The 'influx secret' command manages secrets.
+menu:
+  v2_0_ref:
+    name: influx secret
+    parent: influx
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret` command manages secrets.
+
+## Usage
+```
+influx secret [flags]
+influx secret [command]
```

Comment:
should we say influx secret [subcommand]? 

or how about combining flags/commands to options?:
influx secret [options]
## Options
delete
find
update
help

Reply:
The current code block copies the output of the actual CLI help, although I'm fine with updating it to "subcommand." There is a distinct difference between commands/subcommands and flags. Commands and subcommands perform an action; flags modify that action in some way. In the context of CLIs, it's important to distinguish the two.